earth_enterprise/src/fusion/portableglobe/cutter/cgi-bin/common/postgres_manager_wrap.py

```diff
@@ -35,6 +35,7 @@
 
 class PostgresManagerWrap(object):
   DB_PORT = postgres_properties.PostgresProperties().GetPortNumber()
+  DB_HOST = postgres_properties.PostgresProperties().GetHost()
   DB_USER = "geuser"
```

Comment:
Hooking up to an arbitrary and authenticating to a remote database will normally require both a user and password, and both should be set from properties or somewhere...